package/MDAnalysis/core/selection.py

```diff
@@ -979,6 +1012,7 @@ def __init__(self, parser, tokens):
             raise ValueError(
                 "Invalid operator : '{0}' Use one of : '{1}'"
                 "".format(oper, self.ops.keys()))
+        self.value = float(value)
```

Comment:
Wouldn't something like:

```python
opposites = {'<': '>', '>': '<', '<=': '>=', '>=': '<='}
try:
    self.value = float(value)
except ValueError:
    try:
        self.value = float(prop)
    except ValueError as e:
        raise e
    else:
        self.prop = value
        self.oper = opposites.get(oper, oper)
```

allow to have both `x > 5` and `5 < x`?
allow to write